### PR TITLE
Update Infix_DHCP Test Topologies

### PR DESCRIPTION
**Pull request by ahmkar94**
The topology for each Infix DHCP test case has been updated to match the naming conventions used in the test files.

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: test/case/infix_dhcp/dhcp_routes/topology.dot [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-	esep="+20";
+	esep="+100";
 
         node [shape=record, fontname="monospace"];
 	edge [color="cornflowerblue", penwidth="2"];
 
-	host [
-	    label="host | { <tgt> tgt | <data> | data }",
-	    pos="0,12!",
+	router [
```
Comment (mattiaswal) [inline test/case/infix_dhcp/dhcp_routes/topology.dot:9]: This is wrong, it is not a router. it is the host. Keep the name host

**Review by mattiaswal — CHANGES_REQUESTED** [review]

File: test/case/infix_dhcp/dhcp_router/topology.dot [hunk 2]
```diff
@@ -1,24 +1,23 @@
 graph "1x2" {
 	layout="neato";
 	overlap="false";
-	esep="+20";
+	esep="+80";
 
         node [shape=record, fontname="monospace"];
 	edge [color="cornflowerblue", penwidth="2"];
 
-	host [
```
Comment (mattiaswal) [inline test/case/infix_dhcp/dhcp_router/topology.dot:9]: This is wrong, it is not a router. it is the host. Keep the name host

File: test/case/infix_dhcp/dhcp_basic/topology.dot [hunk 3]
```diff
@@ -1,24 +1,23 @@
 graph "1x2" {
 	layout="neato";
 	overlap="false";
-	esep="+20";
+	esep="+80";
 
         node [shape=record, fontname="monospace"];
 	edge [color="cornflowerblue", penwidth="2"];
 
-	host [
```
Comment (mattiaswal) [inline test/case/infix_dhcp/dhcp_basic/topology.dot:9]: This is also a little strange, it is still the host. I think we should stick with calling it host.

Comment (ahmkar94) [reply test/case/infix_dhcp/dhcp_basic/topology.dot:9]: yes, I agree. I'll revert to 'host' :)

**Review by mattiaswal — APPROVED** [review]

**Review by mattiaswal — APPROVED** [review]

**Pull request merged**